scripts/test-upgrade.sh

```diff
@@ -43,5 +43,6 @@ rm /etc/openstack_deploy/user_extra_variables.yml /etc/openstack_deploy/user_var
 #        OSA upgrade script in liberty, but is not done in the OSA mitaka
 #        upgrade script. Thus, we destroy the containers here.
 cd ${OA_DIR}/playbooks
+openstack-ansible rpc_pre_upgrade.yml
```

Comment:
The directory is wrong for this playbook. Also note that the other code includes comments to explain it's inclusion.